wca-guidelines.md

```diff
@@ -172,7 +172,7 @@ To be more informative, each Guideline is classified using one of the following
 - A6e+) [CLARIFICATION] If the puzzle falls into the lap of the competitor after stopping the timer, this is considered touching the puzzle.
 - A6g+) [ADDITION] While the judge is determining whether to assign a penalty for misalignment, they should not touch the puzzle.
 - A7c+) [ADDITION] If a result signed by a competitor is found to be incomplete or unreadable, it should be interpreted the worst reasonable way (e.g. "1:05" will be considered 1:05.99, "25.X3" will be considered 25.73 if X could be either a 1 or a 7). Given that the Stackmat timer displays a time of X:0Y.ss as X:Y.ss, a missing tens digit should be considered more likely than a missing ones digit (e.g. "1:2.27" will be considered 1:02.27). The judge who signed an ambiguous result may be requested to clarify it, at the discretion of the score taker.
-- A7c4+) [ADDITION] Missing signatures may be rectified if the responsible judge and/or competitor can be tracked, at the discretion of the score taker.
+- A7c4+) [ADDITION] Missing signatures found by the score taker may be added by the judge and/or competitor before the end of the competition, at the discretion of the Delegate.
```

Comment:
For consistency, it is "WCA Delegate" everywhere else in regs.

Reply:
Thanks, I've fixed that!